Makefile

```diff
@@ -106,7 +106,13 @@ $(ARTIFACTS):
 
 .PHONY: test
 test: ## Run tests
-	go test -v ./...
+	source ./scripts/fetch_ext_bins.sh; fetch_tools; setup_envs; go test -v ./...
```

Comment:
@Xenwar Can you maybe drop the -v in the test target and add a test-verbose (analog to the cluster-api repo)

Reply:
Thanks. Removed